Replaces a name from mozc-pinyin to pinyin for a compatibility.

ChromeOS is published on China, so it is not a good idea to replace "pinyin"
tag with "mozc-pinyin" tag because ibus-pinyin user must reconfigure ChormeOS
to use ibus-mozc-pinyin. To avoid this issue, we simply use "pinyin" tag on
ChromeOS.

diff -urN ../mozc_origin/languages/pinyin/unix/ibus/main.h ./languages/pinyin/unix/ibus/main.h
--- ../mozc_origin/languages/pinyin/unix/ibus/main.h	2012-04-09 12:28:03.895173124 +0900
+++ ./languages/pinyin/unix/ibus/main.h	2012-04-09 15:36:18.533924555 +0900
@@ -48,9 +48,11 @@
   "us",
   "us(dvorak)",
 };
+// Using "pinyin*" instead of "mozc-pinyin*" for a compatibility with
+// ibus-pinyin on ChromeOS.
 const char *kEngineNameArray[] = {
-  "mozc-pinyin",
-  "mozc-pinyin-dv",
+  "pinyin",
+  "pinyin-dv",
 };
 const char *kEngineLongnameArray[] = {
   "Mozc Pinyin",
diff -urN ../mozc_origin/languages/pinyin/unix/ibus/mozc-pinyin.xml ./languages/pinyin/unix/ibus/mozc-pinyin.xml
--- ../mozc_origin/languages/pinyin/unix/ibus/mozc-pinyin.xml	2012-04-09 12:28:03.895173124 +0900
+++ ./languages/pinyin/unix/ibus/mozc-pinyin.xml	2012-04-09 15:36:45.563847591 +0900
@@ -14,7 +14,9 @@
       <language>zh-CN</language>
       <icon>/usr/share/ibus-mozc-pinyin/product_icon.png</icon>
       <layout>us</layout>
-      <name>mozc-pinyin</name>
+      <!-- Using "pinyin" instead of "mozc-pinyin" for a compatibility
+	   with ibus-pinyin on ChromeOS -->
+      <name>pinyin</name>
       <longname>Mozc Pinyin</longname>
     </engine>
     <engine>
@@ -23,7 +25,9 @@
       <language>zh-CN</language>
       <icon>/usr/share/ibus-mozc-pinyin/product_icon.png</icon>
       <layout>us(dvorak)</layout>
-      <name>mozc-pinyin-dv</name>
+      <!-- Using "pinyin-dv" instead of "mozc-pinyin-dv" for a compatibility
+	   with ibus-pinyin on ChromeOS -->
+      <name>pinyin-dv</name>
       <longname>Mozc Pinyin (for US Dvorak keyboard)</longname>
     </engine>
   </engines>
